wsgi.py

```diff
@@ -9,17 +9,14 @@
 
 app = Flask(__name__)
 
-@app.route('/weather/api/v1/<city>&<state>', methods=['GET'])
+@app.route('/weather/api/v1/<lat>&<long>', methods=['GET'])
```

Comment:
I like the change here from city to LAT and LONG, much more precise way of making that API call.